app/experimenter/static/css/experimenter.css

```diff
@@ -186,3 +186,19 @@ pre.json {
   line-height: 2em;
   border: none !important;
 }
+
+.list-summary:before {
+  content:'';
+  width:100%;
+  height:100%;    
+  position:absolute;
+  left:0;
+  top:0;
+  background:linear-gradient(transparent 12em, white);
+}
+
+.list-summary{
+  overflow: hidden;
+  max-height: 15em;
+  
+}
```

Comment:
this is a total nit but should there not be an extra blank line here?

Reply:
you're right. lemme fix that....